src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, radius = " + radius + " units, area = "
```

Comment:
You can use it instead. But you can leave "circle" as well, it's ok
```suggestion
        System.out.println("Figure: " + getClass().getSimpleName() + ", radius = " + radius + " units, area = "
```